buildlib/cbuild

```diff
@@ -548,6 +548,11 @@ os.symlink({tarfn!r},os.path.join(b"SOURCES",tarfn));
 
             extra_opts = getattr(env,"rpmbuild_options", [])
             bopts = ["-bb",tspec_file] + extra_opts;
+            for arg in args.with_flags:
+                bopts.extend(["--with", arg]);
+            for arg in args.without_flags:
+                bopts.extend(["--without", arg]);
+
```

Comment:
This extra blank line seems unnecessary..

Reply:
Fixed